diff --git a/.env.local.example b/.env.local.example
deleted file mode 100644
index 46dc59c..0000000
--- a/.env.local.example
+++ /dev/null
@@ -1,6 +0,0 @@
-NEXT_PUBLIC_SANITY_PROJECT_ID=
-NEXT_PUBLIC_SANITY_DATASET=production
-SANITY_REVALIDATE_SECRET=
-
-# for usage in localhost:3333
-SANITY_STUDIO_PROJECT_ID=
\ No newline at end of file
diff --git a/README.md b/README.md
index 8d3cfde..e2bf714 100644
--- a/README.md
+++ b/README.md
@@ -1,6 +1,6 @@
-# Stablo Blog Template - Next.js & Sanity CMS
+## Cheatsheets
+* GROQ: <https://www.sanity.io/docs/query-cheat-sheet>
 
-Stablo is a JAMStack Blog template built with Next.js, Tailwind CSS & Sanity CMS by [Web3Templates](https://web3templates.com/). It comes with free & pro version.
 
 ## [Installation ↓](#installation)
 
@@ -10,44 +10,6 @@ Stablo is a JAMStack Blog template built with Next.js, Tailwind CSS & Sanity CMS
 
 **[Pro Version Demo →](https://stablo-pro.web3templates.com/)**
 
-## Features
-
-<!-- prettier-ignore -->
-| Feature | Free Version | Pro Version |
-| --- | ------ | --- |
-| Next.js v13 | ✅  | ✅ |
-| /app Directory | ✅  | ✅ |
-| Tailwind CSS   | ✅  | ✅ |
-| Sanity CMS (v3) | ✅  | ✅ |
-| Mobile Responsive | ✅  | ✅ |
-| Dark & Light Mode | ✅  | ✅ |
-| Working Contact Page | ✅  | ✅ |
-| Archive (Pagination) | ✅  | ✅ |
-| Category Pages  | ❌  | ✅ |
-| Author Pages    | ❌  | ✅ |
-| Search Page     | ❌  | ✅ |
-| Homepage - Default | ✅  | ✅ |
-| Homepage - Alternate       | ❌  | ✅ |
-| Homepage - Minimal | ❌  | ✅ |
-| Homepage - Lifestlye       | ❌  | ✅ |
-| Homepage - Two Column      | ❌  | ✅ |
-| Blog Post - Default        | ✅  | ✅ |
-| Blog Post - Minimal        | ❌  | ✅ |
-| Blog Post - Lifestlye      | ❌  | ✅ |
-| Blog Post - with Sidebar   | ❌  | ✅ |
-| 6 Months Support| ❌  | ✅  |
-| Free Updates    | ✅  | ✅  |
-| License         | GPL-2.0 | Commercial |
-| &nbsp; | &nbsp;| &nbsp;|
-| Pricing| Free|**$49**|
-| &nbsp; | [Deploy for free](https://vercel.com/new/web3templates/clone?demo-title=Stablo%20%E2%80%93%20Minimal%20Blog%20Template&demo-description=A%20minimal%20blog%20website%20template%20built%20with%20Next.js%2C%20TailwindCSS%20%26%20Sanity%20CMS%0A&demo-url=https%3A%2F%2Fstablo-template.vercel.app%2F&demo-image=%2F%2Fimages.ctfassets.net%2Fe5382hct74si%2F6p72KDrdJ8SjyvOBrgRbnr%2F0760db43f2cb08504a2f67601e74d986%2FCleanShot_2022-07-15_at_16.54.17.png&project-name=Stablo%20%E2%80%93%20Minimal%20Blog%20Template&repository-name=stablo-blog&repository-url=https%3A%2F%2Fgithub.com%2Fweb3templates%2Fstablo&from=templates&integration-ids=oac_hb2LITYajhRQ0i4QznmKH7gx) | [Purchase Pro](https://web3templates.com/templates/stablo-minimal-blog-website-template) |
-
-<a href="https://web3templates.com/templates/stablo-minimal-blog-website-template">
-<img width="160" alt="Upgrade to Pro" src="https://user-images.githubusercontent.com/1884712/199181300-37c2128e-d033-4145-a906-16fa5263a53b.png">
-</a>
-
-###### Click the above button for one-click clone & deploy for this template. Read [quick start](#quick-start) guide below.
-
 #### Template Preview
 
 [![Next.js Front-end Preview](https://user-images.githubusercontent.com/1884712/169838344-e32b7426-621a-45a4-aba8-afedf3377e1f.jpeg)](https://web3templates.com/preview/stablo)
diff --git a/app/(website)/about/about.js b/app/(website)/about/about.js
deleted file mode 100644
index 05f6984..0000000
--- a/app/(website)/about/about.js
+++ /dev/null
@@ -1,55 +0,0 @@
-import Container from "@/components/container";
-import { urlForImage } from "@/lib/sanity/image";
-import Image from "next/image";
-import Link from "next/link";
-
-export default function About({ authors, settings }) {
-  return (
-    <Container>
-      <h1 className="mt-2 mb-3 text-3xl font-semibold tracking-tight text-center lg:leading-snug text-brand-primary lg:text-4xl dark:text-white">
-        About
-      </h1>
-      <div className="text-center">
-        <p className="text-lg">We are a small passionate team.</p>
-      </div>
-
-      <div className="grid grid-cols-3 gap-5 mt-6 mb-16 md:mt-16 md:mb-32 md:gap-16">
-        {authors.slice(0, 3).map(author => {
-          const imageProps = urlForImage(author?.image) || null;
-          return (
-            <div
-              key={author._id}
-              className="relative overflow-hidden rounded-md aspect-square odd:translate-y-10 odd:md:translate-y-16">
-              <Link href={`/author/${author.slug}`}>
-                <Image
-                  src={imageProps.src}
-                  alt={author.name || " "}
-                  fill
-                  sizes="(max-width: 320px) 100vw, 320px"
-                  className="object-cover"
-                />
-              </Link>
-            </div>
-          );
-        })}
-      </div>
-
-      <div className="mx-auto prose text-center dark:prose-invert mt-14">
-        <p>
-          We provide real-time connectivity to enable software
-          providers and financial institutions to build integrated
-          products for their small business customers.
-        </p>
-        <p>
-          Our API infrastructure is leveraged by clients ranging from
-          lenders to corporate card providers and business forecasting
-          tools, with use cases including automatic reconciliation,
-          business dashboarding, and loan decisioning.
-        </p>
-        <p>
-          <Link href="/contact">Get in touch</Link>
-        </p>
-      </div>
-    </Container>
-  );
-}
diff --git a/lib/sanity/client.ts b/lib/sanity/client.ts
index 91d1a54..9ebe971 100644
--- a/lib/sanity/client.ts
+++ b/lib/sanity/client.ts
@@ -6,7 +6,7 @@ import {
   configQuery,
   singlequery,
   pathquery,
-  allauthorsquery,
+  allproducersquery,
   authorsquery,
   postsbyauthorquery,
   postsbycatquery,
@@ -91,7 +91,7 @@ export async function getAuthorPostsBySlug(slug) {
 
 export async function getAllAuthors() {
   if (client) {
-    return (await client.fetch(allauthorsquery)) || [];
+    return (await client.fetch(allproducersquery)) || [];
   }
   return [];
 }
diff --git a/lib/sanity/groq.js b/lib/sanity/groq.js
index efafa9c..ff860ee 100644
--- a/lib/sanity/groq.js
+++ b/lib/sanity/groq.js
@@ -123,9 +123,9 @@ export const searchquery = groq`*[_type == "post" && _score > 0]
    slug
 }`;
 
-// Get all Authors
-export const allauthorsquery = groq`
-*[_type == "author"] {
+// Get all Producers
+export const allproducersquery = groq`
+*[_type == "producer"] {
  ...,
  'slug': slug.current,
 }
diff --git a/lib/sanity/schemas/author.js b/lib/sanity/schemas/author.js
deleted file mode 100644
index 82ad458..0000000
--- a/lib/sanity/schemas/author.js
+++ /dev/null
@@ -1,48 +0,0 @@
-export default {
-  name: 'author',
-  title: 'Author',
-  type: 'document',
-  fields: [
-    {
-      name: 'name',
-      title: 'Name',
-      type: 'string',
-    },
-    {
-      name: 'slug',
-      title: 'Slug',
-      type: 'slug',
-      options: {
-        source: 'name',
-        maxLength: 96,
-      },
-    },
-    {
-      name: 'image',
-      title: 'Image',
-      type: 'image',
-      options: {
-        hotspot: true,
-      },
-    },
-    {
-      name: 'bio',
-      title: 'Bio',
-      type: 'array',
-      of: [
-        {
-          title: 'Block',
-          type: 'block',
-          styles: [{title: 'Normal', value: 'normal'}],
-          lists: [],
-        },
-      ],
-    },
-  ],
-  preview: {
-    select: {
-      title: 'name',
-      media: 'image',
-    },
-  },
-}
diff --git a/lib/sanity/schemas/index.ts b/lib/sanity/schemas/index.ts
index 7cd3948..e1dea60 100644
--- a/lib/sanity/schemas/index.ts
+++ b/lib/sanity/schemas/index.ts
@@ -1,7 +1,8 @@
-import blockContent from './blockContent'
-import category from './category'
-import post from './post'
-import author from './author'
-import settings from './settings'
+import blockContent from './blockContent';
+import category from './category';
+import post from './post';
+import producer from './producer';
+import settings from './settings';
+import item from './item';
 
-export const schemaTypes = [post, author, category, settings, blockContent]
+export const schemaTypes = [item, post, producer, category, settings, blockContent]
diff --git a/lib/sanity/schemas/post.js b/lib/sanity/schemas/post.js
index 8b15d37..c6015fe 100644
--- a/lib/sanity/schemas/post.js
+++ b/lib/sanity/schemas/post.js
@@ -20,21 +20,6 @@ export default {
         maxLength: 96
       }
     },
-    {
-      name: "excerpt",
-      title: "Excerpt",
-      description:
-        "The excerpt is used in blog feeds, and also for search results",
-      type: "text",
-      rows: 3,
-      validation: Rule => Rule.max(200)
-    },
-    {
-      name: "author",
-      title: "Author",
-      type: "reference",
-      to: { type: "author" }
-    },
     {
       name: "mainImage",
       title: "Main image",
